src/components/LoginDialog/LoginDialog.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <v-dialog v-model="dialog" width="500">
+  <v-dialog
```

Comment:
We're using this kind of dialog in several places (one that is fullscreen on mobile), perhaps we can create a component out of it and reuse it across the codebase. What do you think?